cibuildwheel/docker_container.py

```diff
@@ -30,6 +30,9 @@ class DockerContainer:
     bash_stdout: IO[bytes]
 
     def __init__(self, docker_image: str, simulate_32_bit: bool = False, cwd: Optional[PathOrStr] = None):
+        if not docker_image:
```

Comment:
Oh, I was expecting it in `__main__`, where we're reading the docker image. But maybe this makes even more sense.